backend/schemas/source/excel/libs/SheetValidations.libsonnet

```diff
@@ -91,4 +99,13 @@ local ComplianceRequirementValidation = {
   YoNValidation: YoNValidation,
   ReferenceNumberValidation: ReferenceNumberValidation,
   ComplianceRequirementValidation: ComplianceRequirementValidation,
+  LoanBalanceValidation: LoanBalanceValidation,
+  AuditReportTypeValidation(formRef): {
+    type: 'list',
+    allow_blank: 'True', 
+    // FIXME MSHD: for improvement, will need to pull column refs from this formula and retrieve that dynamically.
+    formula1: '=IF(R{0}="Y",' + formRef + '!$A$2:$A$5,' + formRef + '!$A$6:$A$6)', 
```

Comment:
Another place for a named range?

Reply:
@jadudm This is the same code you commented on earlier. Actually, what you commented on earlier was the output generated jsonnet whereas this code is the source. What I did was to parameterize the formula, treating it like a function. This allows me to pass a value for "formRef" wherever I need to utilize that formula (in the xlsx template). But, I agree with your suggestion. A named range would be simpler in this case.